docs/qcfractal/source/changelog.rst

```diff
@@ -15,15 +15,61 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
-0.6.1 / 2019-??-??
+0.7.0 / 2019-05-??
 ------------------
 
```

Comment:
Can we add a big warning here that this will be the last Mongo compatible release?